Bio/PDB/cealign.py

```diff
@@ -86,7 +90,7 @@ def set_reference(self, structure):
             )
             raise PDBException(msg)
 
-    def align(self, structure, transform=True):
+    def align(self, structure, transform=True, *, final_optimization=True):
```

Comment:
Why the `*`? There were no additional arguments here so this is a safe add?

Reply:
The `*` makes the `final_optimization` parameter a keyword-only parameter (defined [here](https://docs.python.org/3/glossary.html#term-parameter)). I think it makes the code more clear if the caller has to do `aligner.align(structure, final_optimization=False)`.

And yes, because I am adding a new argument, the change does not break anyone's code.

Reply:
Oh gotcha, so that people won't do `align(struct, True True)`. Alright, makes sense!